meinberlin/apps/budgeting/assets/__tests__/BudgetingProposalList.jest.jsx

```diff
@@ -1,42 +1,80 @@
 import React from 'react'
-import { render } from '@testing-library/react'
+import { render, act, screen } from '@testing-library/react'
 import { BudgetingProposalList } from '../BudgetingProposalList'
 
-// const data = {results: [
-//   {
-//     name: 'myProposal',
-//     url: 'www',
-//     budget: '80639',
-//     creator: 'creator',
-//     created: '2021-11-11T15:37:19.490201+01:00'
-//   },
-//   {
-//     name: 'This is a second proposal',
-//     url: 'www',
-//     budget: '12000',
-//     creator: 'creator part 2',
-//     created: '2021-10-29T09:33:36.280033+01:00'
-//   }
-// ]}
-
-// Mocks the fetch request
-global.fetch = jest.fn(() =>
-  Promise.resolve({
-    json: () => Promise.resolve({ data: {} })
+test('Budgeting Proposal List without list item (empty)', async () => {
```

Comment:
@phillimorland finally i found a solution for the `...a test was not wrapped in act(...).` we had with the test of `BudgetingProposalList.jsx` . 

Source: https://kentcdodds.com/blog/fix-the-not-wrapped-in-act-warning#the-dreaded-act-warning

Three cases are testing the list here:
- If no item was fetched
- If one (or more) items are fetched
- If a network error occured, while fetching

With these tests I think the snapshot of the BudgetingList is not needed anymore. So I deleted it. Hope that is fine.